providers/flagd/README.md

```diff
@@ -105,23 +89,28 @@ automatically.
 
 ### Deadline (gRPC call timeout)
 
-The deadline for an individual flag evaluation can be configured by calling `setDeadline(< deadline in millis >)`.
+The deadline for an individual flag evaluation can be configured by calling `setDeadline(myDeadlineMillis)`.
 If the gRPC call is not completed within this deadline, the gRPC call is terminated with the error `DEADLINE_EXCEEDED`
 and the evaluation will default.
 The default deadline is 500ms, though evaluations typically take on the order of 10ms.
+For the in-process provider, the deadline is used when establishing the initial streaming connection.
+A failure to connect within this timeout will result an [error event](https://openfeature.dev/docs/reference/concepts/events#provider_error) from the provider, though it will attempt to reconnect indefinitely.
 
 ### TLS
 
-Though not required in deployments where flagd runs on the same host as the workload, TLS is available.
+Though likely unnecessary in deployments where flagd runs on the same host as the workload, TLS is available.
```

Comment:
Perhaps we should reverse this statement and say something like `TLS is available in situations where flagd is running on another host`. Adding a code snippet would be nice as well.

Reply:
I guess the issue I have with that wording is that it's available in the case when flagd is running on the same host as well... it's just a bit silly to use it there

Reply:
I added this phrasing, added a code snippet, and also mentioned that you can use a x509 cert in pem format.